Makefile

```diff
@@ -45,7 +46,12 @@ ci-image: .build-docker
 check-ceph-version:
 	@grep -wq '$(CEPH_VERSION)' .build-docker 2>/dev/null || $(RM) .build-docker
 
-check:
+check: check-revive check-format
```

Comment:
is it reasonable to expect all developers to have `revive` installed?

Reply:
If they want to run make check locally, yeah. That's the tool we use for static checks. :-)
I'm totally open to doing some sort of `make check-containerized` in the future.